modules/end-user-guide/pages/url-parameter-for-container-image.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: CONCEPT
+:description: URL parameter for a container image
+:keywords: parameter, URL, container, image
+:navtitle: URL parameter for a container image
+//:page-aliases:
+
+[id="url-parameter-for-container-image"]
+= URL parameter for a container image
+
+You can use the `image` parameter to add a path to a container image in the following scenarios:
+
+* The Git repository contains no devfile, and you want to start a new workspace with this container.
+
+* The Git repository contains a devfile, and you want this container image to run instead of the first `container` `image` listed in the `components` section of the devfile.
+
+The URL parameter for the path to a container image is `image=`:
+
+[subs="+quotes"]
+----
+https://<che_fqdn>#<git_repository_url>?image=<container_image_url>
+----
```

Comment:
We have an attribute for that. You can just reuse bits from other pages. I've just had a look at another similar page:
https://github.com/eclipse-che/che-docs/blob/main/modules/end-user-guide/pages/url-parameter-for-the-ide.adoc?plain=1#L22
```suggestion
[source,subs="+quotes,+attributes,+macros"]
----
pass:c,a,q[{prod-url}]#<git_repository_url>?image=<container_image_url>
----
```